src/services/UserService.ts

```diff
@@ -1 +1,20 @@
-export {}
+import AuthHelper from '../helpers/AuthHelper'
+import { UserInterface } from '../interfaces/UserInterface'
+import UserRepository from '../repositories/UserRepository'
+import GenericError from '../helpers/GenericError'
+
+const createUser = async ({ name, email, password }: UserInterface) => {
```

Comment:
If you want to use the same interface, make sure to omit the `id` from the interface
```
const createUser = async ({ name, email, password }: Omit<UserInterface, id>) => 
```
Or you can create a new interface.